libs/core/src/lib/bar/bar.component.ts

```diff
@@ -17,59 +17,40 @@ export type BarDesignType = 'header' | 'subheader' | 'header-with-subheader' | '
     encapsulation: ViewEncapsulation.None,
     changeDetection: ChangeDetectionStrategy.OnPush
 })
-export class BarComponent implements OnInit, OnChanges, CssClassBuilder {
+export class BarComponent implements OnChanges, OnInit, CssClassBuilder {
 
     /** user's custom classes */
-    private _class: string = '';
     @Input()
-    set class(userClass: string) {
-        this._class = userClass;
-        this.buildComponentCssClass();
-    }
+    class: string;
```

Comment:
I believe this changes were introduced in https://github.com/SAP/fundamental-ngx/pull/2228

Reply:
I actually merged that branch in to this branch as this branch depends on it.  So that is why you are seeing all the changes to a bunch of different files